tests-lit/tools/FileCheck.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python3
```

Comment:
@AlexDenisov Are you comfortable with having FileCheck.py as source file, not as Python package for now?

Reply:
Another option is to keep the binaries for now. Just let me know what you think.

Reply:
Python source is definitely better than a binary.